msg/SensorUwb.msg

```diff
@@ -0,0 +1,34 @@
+# UWB distance contains the distance information measured by an ultra-wideband positioning system,
+# such as Pozyx or NXP Rddrone.
+
+uint64 	timestamp		# time since system start (microseconds)
+
+uint32 	sessionid		# UWB SessionID
+uint32 	time_offset		# Time between Ranging Rounds in ms
+uint32 	counter			# Number of Ranges since last Start of Ranging
+uint16 	mac			# MAC adress of Initiator (controller)
+
+uint16 	mac_dest		# MAC adress of Responder (Controlee)
+uint16 	status			# status feedback #
+uint8 	nlos			# None line of site condition y/n
+float32 distance		# distance in m to the UWB receiver
+
+
+#Angle of arrival, Angle in Degree -60..+60; FOV in both axis is 120 degrees
```

Comment:
Why is it called Angle of Arrival?
Isn't it just the current angle relative to the target? To me "Angle of Arrival" sounds like a setpoint, a desired angle of arrival.

Is this related to moving targets? Standard precision landing on a stationary target just lands with a zero horizontal velocity, so the angle would always be 90 degrees.

Reply:
UWB measures the Angle of Arriving signals in a 120° FOV infront of the antennas.

Reply:
Maybe a picture in the PR description would help for the uninitiated :sweat_smile:

Reply:
In general I think we will have to rework this at some point, such that the landing_target estimator does not know about IR beacons, UWB, or other technologies.
The landing target estimator should receive coordinates of a target in a unified format that is the same for all sensor types. 

So far I'm not too concerned because the logic for UWB is cleanly isolated in its own `else {}` block.

Reply:
I agree.
I do not think it is the correct way to have the physical characteristics of the IRlock or the UWB sensor inside the LTEST module. They should be set at driver level and then read from there.
Atm we abuse UORB to send these params for UWB but with an update rate of up to 200hz this should be streamlined.

Also this whole Landing_target_estimator should probably go. 
As you mentioned it should be one target estimator that can then use information from multiple sensors.
Did you see #20653 which tries to improve the situation.